challenges-webapp/src/main/webapp/vue-app/challenges/components/WelcomeMessage.vue

```diff
@@ -2,12 +2,13 @@
   <div class="welcomeMessage">
     <img
       src="/challenges/images/Challenge_app_backg_image.png"
+      alt="challenges"
```

Comment:
please add description for this img and use i18n

You can see more details here : https://www.semrush.com/blog/alt-text/